index.html

```diff
@@ -173,6 +173,17 @@ <h2 id="about-title">About me</h2>
         </ul>
       </div>
     </section>
+
+    <section id="contact">
+      <p>I'm always interested in hearing about new projects, so if you'd like to chat please do get in touch!</p>
+      <form id="contact-form" action="https://formspree.io/f/xwkydlkr" method="post">
+        <input name="user_name" id="name" type="text" placeholder="Full Name" required />
+        <input name="user_email" id="email" type="email" placeholder="Email address" required />
+        <textarea name="user_msg" id="msg" placeholder="Enter text here" required></textarea>
```

Comment:
Kindly add HTML validations as stated in project requirements to make your form inputs properly monitored. example: ( maxlength="30" on name input)